c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -1247,7 +1247,7 @@ void MfxC2EncoderComponent::DoWork(std::unique_ptr<C2Work>&& work)
             native_handle_t *grallocHandle = android::UnwrapNativeCodec2GrallocHandle(c_graph_block->handle());
             // From Android U, the get function of IMapper4 will check whether the buffer handle is reserved.
             // So we need to call importBuffer before getting the buffer's info.
-#if MFX_ANDROID_VERSION >= MFX_U
+#if PLATFORM_SDK_VERSION >= 33
```

Comment:
Is it better we add comment indicating Android 14(U) (API level 34 instead of 33) ? It is easy to read. Just a suggestion.

https://developer.android.com/tools/releases/platforms#14

Reply:
yes, sure, thanks!